unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWPublishShould.cs

```diff
@@ -81,6 +81,16 @@ public void TestMetricsPublish()
         Assert.IsFalse(biwPublishController.CanPublish());
     }
 
+    [Test]
+    public void TestPublishFeedbackMesasge()
```

Comment:
```suggestion
    public void TestPublishFeedbackMessage()
```
Please please be more careful with those typos, this PR is littered with them.